### ISSUE TITLE
User Account Signup/Creation

### ISSUE THREAD

**Issue by skukuk1** [it-sd/sqc-project-skukuk1#4]
I would potentially like to have a spot where someone can add details and create an account. Basically, a signup page that might connect to the database in the future. Details would include their name, email, phone number, interests, and other fields that might be interesting to have on a profile.

**Comment by LeviJLiljedahl** [comment]
Hello,
I really like your website design and the overall idea behind it, so I went ahead and tackled this issue. You will be able to see the [pull request](https://github.com/it-sd/sqc-project-skukuk1/pull/7) on you main dashboard. I created a functioning create account page, with some basic validation that, upon submit, saves the entered information into your database, via a new table I created in your schema called 'account'. I hope this is what you had in mind and I hope you like it. If you have any questions or concerns, feel free to reach out at lmarquardt7@student.cvtc.edu. Thanks!
